### DISCUSSION TITLE
2024.03.28 - Community Meeting

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#9359] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfil this, we would like to held Bimonthly Community Meeting where we can share main updates and also have an open discussion with you.

---

**💥 Fifteenth Community Meeting will take place on Thursday 28/03/2024 at 4 pm GMT+1 time zone. (1 hour length) We will use Google Meet.**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you already participated on any Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you join us again or not.

---

### Agenda will be specified below in the comments:  (feel free to add any topic or idea)

1) Development updates & plans

2) Community Topics

3) Open Discussion

---

See you on the Meeting!

**Comment by VojtechBartoska** [comment]
Hello, Community Meeting will take place this Thursday, feel free to add any topics suggestions 👍

**Comment by PaulaScharf** [comment]
Hi, could you perhaps share the slides?

**Reply by VojtechBartoska** [reply]
Hello @PaulaScharf, sure! Here we go.

[15_Community_meeting-Arduino-March_2024.pdf](https://github.com/espressif/arduino-esp32/files/15012729/15_Community_meeting-Arduino-March_2024.pdf)
